NEWS.md

```diff
@@ -52,8 +67,11 @@
 * Fix to prevent sending refresh complete notification if snap snap-refresh-observe interface is connected
 * Fix to queue snapctl service commands if run from the default-configure hook to ensure they get up-to-date config values
 * Fix stop service failure when the service is not actually running anymore
+* Fix parsing /proc/PID/mounts with spaces
 * Add registry interface that provides snaps access to a particular registry view
-* steam-support interface: relaxed AppArmor and seccomp restrictions to improve user experience
+* Add snap-interfaces-requests-control interface to enable prompting client snaps
```

Comment:
this is redundant with line 16 above, I think: https://github.com/canonical/snapd/pull/14397/files#diff-51920e95310ebfbc1ae31709f3b95f89afffbf4f1a6e38e8b2b406e2fb6197eaR16

Not sure whether to keep both or just one or the other. I think the part about the handler service on line 16 is important for context for line 18.

Reply:
Thanks, will fix

Reply:
fixed

Reply:
fixed